packages/aws-cdk/lib/init.ts

```diff
@@ -153,10 +153,14 @@ export class InitTemplate {
 
   private async installProcessed(templatePath: string, toFile: string, language: string, project: ProjectInfo) {
     const template = await fs.readFile(templatePath, { encoding: 'utf-8' });
-    await fs.writeFile(toFile, this.expand(template, language, project));
+    let appCommandTool = 'npx';
+    if (toFile.includes('cdk.json') && await getCommand() === 'pnpm') {
```

Comment:
This is hard-coded for pnpm, but we wont support yarn or other package managers etc. Is that the approach we want?

Reply:
per my understanding we do not want to extend this feature pass `pnpm` following the comments in the original issue I think we should just expand the support to `pnpm`. Otherwise we expose ourselves into making changes every time there is a possibility of having a new package manager. Any thoughts?